<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Typing Speed Tester</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="highscore_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>170</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="highscore_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Highscores:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="all_time_highscore">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>80</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00 wpm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="today_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>45</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Today:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="today_highscore">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00 wpm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="all_time_label">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>80</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>Can be reset by deleting All_Time.txt</string>
     </property>
     <property name="text">
      <string>All time:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="difficulty_frame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="begin_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="statusTip">
      <string>Pops up a new window with the actual typing test</string>
     </property>
     <property name="text">
      <string>Begin Speed Test</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hard_radio">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>45</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="statusTip">
      <string>Type out long paragraphs, either paragraphs from famous books or famous quotes</string>
     </property>
     <property name="text">
      <string>Hard</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="medium_radio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>45</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="statusTip">
      <string>Type some common English expressions/sayings</string>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="easy_radio">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>45</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="statusTip">
      <string>Type some of the most common English words</string>
     </property>
     <property name="text">
      <string>Easy</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="choose_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Choose a difficulty</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
